app/src/components/panels/helpPanel/helpPanel.scss

```diff
@@ -0,0 +1,61 @@
+.help-panel-container {
```

Comment:
For all new files we add
/*
 * Copyright 2024 EPAM Systems
 * ....
 * limitations under the License.
 * /

Reply:
thanks for mentioning!